.github/workflows/podPublish.yml

```diff
@@ -1,12 +1,14 @@
 name: Pod-Publish
 on:
   push:
+    branches:
+      - main
+      - main_0.1
     tags:
       - 0.[0-9]+.[0-9]+ # Run automatically on new tags matching a 0.N.M pattern
```

Comment:
Would this match the tags we've been using for `main_0.1`? The latest tag was `0.1.34_main_0.1`. I'm not sure how strict this is regarding the start/end of the string.

If it works then I'm all for it!

Reply:
oh I didn't know we had a different tagging system for main_0.1 branch

Reply:
Yeah I'm not sure why; @rdeassis might have more background of why it is a different pattern.

Reply:
If I remember correctly, we didn't want this to be automatic by accident, which is why we created the different tagging pattern. It'd be an extra layer of safety combined with the branches listed.

Reply:
handled!